native/SampleApps/RestAPIExplorer/RestAPIExplorer/AppDelegate.swift

```diff
@@ -167,9 +149,9 @@ class AppDelegate : UIResponder, UIApplicationDelegate
                 return
         }
         
-        var config = ["test_client_id": SalesforceSwiftSDKManager.shared().appConfig?.remoteAccessConsumerKey,
+        var config = ["test_client_id": SalesforceSDK.shared().appConfig?.remoteAccessConsumerKey,
```

Comment:
Shouldn't this be `SmartSyncSDKManager` (that's what it derives from)?

Reply:
This app doesn't use smartstore or smartsync so they can use the core manager.

Reply:
Oh right - this is `RestAPIExplorer`. My bad.